bin/sidecli.re

```diff
@@ -262,12 +276,9 @@ let info_withdraw = {
   Term.info("withdraw", ~version="%‌%VERSION%%", ~doc, ~exits, ~man);
 };
 
-let withdraw =
-    (folder_node, sender_wallet_file, tezos_address, amount, ticket) => {
+let withdraw = (endpoint, sender_wallet_file, tezos_address, amount, ticket) => {
```

Comment:
I would say to add a `folder_node` like the other commands, especially as the sidecli is mostly target as development.

Reply:
Addressed in the incoming commit